<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QChar常用功能</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>宋体</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>输入字符串</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>宋体</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>输入单个字符</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="editStr">
              <property name="text">
               <string>Hello，青岛</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editChar">
              <property name="text">
               <string>★</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="btnGetChars">
              <property name="font">
               <font>
                <family>新宋体</family>
               </font>
              </property>
              <property name="text">
               <string>每个字符的Unicode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCharJudge">
              <property name="font">
               <font>
                <family>新宋体</family>
               </font>
              </property>
              <property name="text">
               <string>单个字符特性判断</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>其他测试和功能</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btnCovLatin1">
              <property name="font">
               <font>
                <family>新宋体</family>
               </font>
              </property>
              <property name="text">
               <string>与Latina的转换</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCompare">
              <property name="font">
               <font>
                <family>新宋体</family>
               </font>
              </property>
              <property name="text">
               <string>QChar比较和替换</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="btnCovUTF16">
              <property name="font">
               <font>
                <family>新宋体</family>
               </font>
              </property>
              <property name="text">
               <string>与UTF-16的转换</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Clear">
              <property name="font">
               <font>
                <family>新宋体</family>
               </font>
              </property>
              <property name="text">
               <string>清空文本框</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>结果显示</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>QChar特性判断</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="checkDigit">
            <property name="text">
             <string>isDigit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkLetter">
            <property name="text">
             <string>isLetter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkLetterOrNumber">
            <property name="text">
             <string>isLetterOrNumber</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkUpper">
            <property name="text">
             <string>isUpper</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkLower">
            <property name="text">
             <string>isLower</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkMark">
            <property name="text">
             <string>isMark</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkSpace">
            <property name="text">
             <string>isSpace</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkSymbol">
            <property name="text">
             <string>isSymbol</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkPunct">
            <property name="text">
             <string>isPunct</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
